Autopilot/AttitudeManager/FixedWing.c

```diff
@@ -0,0 +1,248 @@
+/*
+ * File:   FixedWing.c
+ * Author: Chris Hajduk
+ *
+ * Created on July 2, 2015, 8:04 PM
+ */
+
+#include "PWM.h"
+#include "AttitudeManager.h"
+#include "delay.h"
+#include "FixedWing.h"
+#include "ProgramStatus.h"
+
+#if VEHICLE_TYPE == FIXED_WING
+
+int outputSignal[NUM_CHANNELS];
+int control_Roll, control_Pitch, control_Yaw, control_Throttle;
+
+void initialization(){
+    setPWM(THROTTLE_OUT_CHANNEL, MIN_PWM);
+    
+    int channel = 0;
+    for (; channel < NUM_CHANNELS; channel++) {
+        outputSignal[channel] = 0;
+    }
+    setProgramStatus(UNARMED);
+
+//    char str[20];
+//    sprintf(str,"AM:%d, PM:%d",sizeof(AMData), sizeof(PMData));
+//    debug(str);
+    while (getProgramStatus() == UNARMED){
+        StateMachine(STATEMACHINE_IDLE);
+    }
+}
+
+void armVehicle(int delayTime){
+    setProgramStatus(ARMING);
+    asm("CLRWDT");
+    Delay(delayTime);
+    asm("CLRWDT");
+    setPWM(THROTTLE_OUT_CHANNEL, MIN_PWM);
+    setPWM(ROLL_OUT_CHANNEL, 0);
+    setPWM(L_TAIL_OUT_CHANNEL, 0);
+    setPWM(R_TAIL_OUT_CHANNEL, 0);
+    setPWM(FLAP_OUT_CHANNEL, MIN_PWM);
+    asm("CLRWDT");
+    Delay(delayTime);
+    asm("CLRWDT");
+}
+
+void dearmVehicle(){
+    int i = 1;
+    for (; i <= NUM_CHANNELS; i++){
+        setPWM(i, MIN_PWM);
+    }
+    setProgramStatus(UNARMED);
+
+    while (getProgramStatus() == UNARMED){
+        StateMachine(STATEMACHINE_IDLE);
+    }
+    setProgramStatus(MAIN_EXECUTION);
+}
+
+void takeOff(){
+
+}
+void landing(){
+    
+}
```

Comment:
Empty functions don't get compiled anyway, yet show the potential for expanded functionality... 

But I see your point... This isn't gonna happen in the near future. 

So yeah remove it I guess... I would accept this PR   even if you had it.